chart/kubeapps/values.yaml

```diff
@@ -496,6 +496,13 @@ dashboard:
   ##      background-color: #991700;
   ##    }
   customStyle: ""
+  ## Custom translations injected to the Dashboard to customize the str
+  ## e.g:
+  ## customLocale: |-
```

Comment:
I would add here a link to the docs (with a note saying that it's not ready yet).

Reply:
Good idea for the moment. However, once this feature is fully-finished, I'd like to add this info to the chart docs, but, for now, I agree with you